### ISSUE TITLE
Port to use glow instead of a function call table

### ISSUE THREAD

**Issue by notgull** [glium/glium#2060]
It would be nice to be able to use this crate on web targets. As far as I know, the only portable way of doing this is to use the [`glow`](https://crates.io/crates/glow) instead of a `get_proc_address` based approach.

**Comment by Melchizedek6809** [comment]
Yeah, supporting web targets in `glium` would be great, do you have time to to look into adding a new `glow`/`webgl` backend to `glium`? I'd love to look over/merge that PR :)

**Comment by notgull** [comment]
This isn't high on my priorities list, so I probably won't get to it in the foreseeable future, sorry

**Comment by Melchizedek6809** [comment]
Ah too bad, on second thought: it might be even better to go one level below and get `glutin` to support web targets, since this would help not only `glium` but many more projects.
